### PR TITLE
IRCCloud SASL instructions

### PR DESCRIPTION
**Pull request by RhinosF1**
Add instructions for using sasl via irccloud.com

### CODE REVIEW

File: content/kb/sasl/irccloud.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+4. Click advanced options
+5. Enter your password in the 'NickServ Password' field
+6. Press save
+---
+Method 2
+---
+
+1. In the channel list, click the settings cog on the server you want to adjust settings for
+2. Click 'identify Nickname'
+3. Enter your passowrd in 'Password for [Nick]
```
Comment (equinox) [inline content/kb/sasl/irccloud.md]:
```suggestion
3. Enter your password in 'Password for [Nick]
```

File: content/kb/sasl/irccloud.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+---
+Method 1
+---
+1. In the channel list, click the settings cog on the server you want to adjust settings for
+2. Click Edit
+
+OR
+
+1. Click on the server to open the server pannel
```
Comment (equinox) [inline content/kb/sasl/irccloud.md]:
```suggestion
1. Click on the server to open the server panel
```

**Comment by RhinosF1** [comment]
@equinox - Done both

**Comment by emersonveenstra** [comment]
@RhinosF1 can you rebase + squash, then we'll merge it?

**Comment by emersonveenstra** [comment]
thanks!

**Pull request merged**